charts/merlin/values.yaml

```diff
@@ -0,0 +1,463 @@
+deployment:
+  image:
+    pullPolicy: IfNotPresent
+    registry: ghcr.io
+    repository: gojek/merlin
+    tag: 0.21.0
+
+  replicaCount: "2"
+
+  # Additional labels to apply to the deployment
+  labels: {}
+
+  # Additional labels to apply to the merlin pods
+  podLabels: {}
+
+  resources:
+    requests:
+      cpu: "500m"
+      memory: 1Gi
```

Comment:
This might cause the runner to run out of resources. Workaround is to specify a separate set of helm values in a `./ci` dir, 
```
Charts may have multiple custom values files matching the glob pattern '*-values.yaml' in a directory named 'ci' in the root of the chart's directory. The chart is installed and tested for each of these files. If no custom values file is present, the chart is installed and tested with defaults.
``` 
from [ct doc](https://github.com/helm/chart-testing/blob/main/doc/ct_install.md)

Reply:
this works. thanks 👍